### PR TITLE
[fix](nereids) avg size of column stats always be 0

### PR DESCRIPTION
**Pull request by englefly**
## Proposed changes
it takes lot of effort to compute the avgSizeByte for col stats. 
we use schema information to avoid compute actual average size. 

Issue Number: close #xxx

<--Describe your changes.-->

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by Kikyou1997 — DISMISSED** [review]
lgtm

File: fe/fe-core/src/main/java/org/apache/doris/nereids/stats/StatsCalculator.java [hunk 1]
```diff
@@ -551,6 +551,11 @@ private Statistics computeScan(Scan scan) {
                 throw new RuntimeException(String.format("Invalid slot: %s", slotReference.getExprId()));
             }
             ColumnStatistic cache = Config.enable_stats ? getColumnStatistic(table, colName) : ColumnStatistic.UNKNOWN;
+            if (cache.avgSizeByte <= 0) {
```
[the remark this answers is not in the document]
Comment (englefly) [reply fe/fe-core/src/main/java/org/apache/doris/nereids/stats/StatsCalculator.java:554]: The avgSizeByte means its size defined in schema. Even for UNKNOWN, its avgSizeByte is available.
And this size is useful when we compute the cost of data transfer, given a schema defined size is better than Zero.

**Review by starocean999 — DISMISSED** [review]

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run buildall

**Comment by englefly** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Pull request merged**